packages/@aws-cdk/aws-servicecatalogappregistry/lib/application-associator.ts

```diff
@@ -28,6 +28,12 @@ export interface ApplicationAssociatorProps {
    */
   readonly description?: string;
 
+  /**
+   * Stack ID.
+   *
+   */
+  readonly stackId: string;
```

Comment:
Can we make this optional instead? In case customers do not want to provide `stackId`, then we can instead default it to `ApplicationAssociator`

Reply:
Thank you for the suggestion! It will also make the change incremental and not breaking, which is great 😃. I pushed the update, although used the previous `ApplicationAssociatorStack` value to avoid introducing a breaking change.